ios/celo/Info.plist

```diff
@@ -53,17 +53,6 @@
 	</array>
 	<key>ITSAppUsesNonExemptEncryption</key>
 	<false/>
-	<key>LSApplicationQueriesSchemes</key>
-	<array>
-		<string>message</string>
-		<string>readdle-spark</string>
-		<string>airmail</string>
-		<string>ms-outlook</string>
-		<string>googlegmail</string>
-		<string>inbox-gmail</string>
-		<string>ymail</string>
-		<string>superhuman</string>
-	</array>
```

Comment:
These were added in https://github.com/celo-org/celo-monorepo/commit/69005a0144dfa52964c74b806d167ed4c7951716 for `react-native-email-link`.